mne_features/utils.py

```diff
@@ -158,3 +165,37 @@ def filt(sfreq, data, filter_freqs, verbose=False):
         return filter_data(data, sfreq=sfreq, l_freq=filter_freqs[0],
                            h_freq=filter_freqs[1], picks=None,
                            fir_design='firwin', verbose=_verbose)
+
+
+def _get_feature_funcs(sfreq, module_name):
+    """ Inspects a given module and returns a dictionary of feature
```

Comment:
pep257

doc should be first 1 short line. THen an empty line and a short paragraph.